### DISCUSSION TITLE
HyperSEQTA

### DISCUSSION THREAD

**Discussion by MEGA-Dawg68** [SethBurkart123/EvenBetterSEQTA#49] [General]
Crazypersonalph made another extension
https://github.com/Crazypersonalph/HyperSEQTA

I honestly think we should all work on one extension, development would be a lot easier and better if we worked together

**Comment by SethBurkart123** [comment]
Yeah, I understand the benefits of all working together on a single extension. Honestly I wish it played out like that, the problem is - I've made a lot of progress and work into this extension, picked up a lot of users and I just don't know that I could back down and leave it all. 

Crazypersonalph is a great developer, so is norangeflame and I wish that we'd all be able to work together. But is it worth me leaving this extension now? With all the work that's been put in. Or what should happen to facilitate the change?

**Comment by MEGA-Dawg68** [comment]
No it isn't worth leaving leaving this extension, but he could definitely leave his, he has under 100 people using it. If we could persuade him to help us that would be good

**Reply by Crazypersonalph** [reply]
"he has under 100 people using it" no need to rub salt into the wounds man 🥲 lol

**Reply by MEGA-Dawg68** [reply]
Sorry 'bout that 😅. Well now you can say you have 5,000 people using your extension 😀 lol

**Comment by SethBurkart123** [comment]
Yeah, maybe we should talk to him in his discord.

**Reply by MEGA-Dawg68** [reply]
good idea

**Reply by SethBurkart123** [reply]
all done! It's decided, we are merging the projects #51

**Reply by MEGA-Dawg68** [reply]
Actual?! That is awesome! Is he going to work on this one as well? or will he continue on his and keep merging?

**Reply by SethBurkart123** [reply]
Well he is a contributor of this project at the moment.

**Comment by SethBurkart123** [comment]
Welp! This one is all decided. We've merged! 🎉
